tools/deb/istio-iptables.sh

```diff
@@ -199,6 +248,7 @@ for uid in ${PROXY_UID}; do
   # Avoid infinite loops. Don't redirect Envoy traffic directly back to
   # Envoy for non-loopback traffic.
   iptables -t nat -A ISTIO_OUTPUT -m owner --uid-owner ${uid} -j RETURN
+  iptables -t nat -A ISTIO_OUTPUT -m owner --gid-owner ${uid} -j RETURN
```

Comment:
Docker (and a fortiori K8s) doesn't set ambient capabilities. And normal capabilities are lost when doing an `execve`. So if we combined `runAsUser` and `capabilities` in a `securityContext`, the `capabilities` would practically be a no-op. Envoy would get no capabilities.
If you wanted `pilot-agent` to pass capabilities to envoy, it would have to be root. And this is exactly what I'm doing here.